justfile

```diff
@@ -17,7 +17,9 @@ setup-env: install-kind create-kind-cluster
 create-vanilla-deployment-file:
    kustomize build {{kustomizeVanillaOverlay}} --output deploy/k8s/cloudbeat-ds.yaml
 
-build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image deploy-cloudbeat
+build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image
```

Comment:
Why is the `deploy-cloudbeat` step removed?

Reply:
Fixed